Emby.Server.Implementations/Localization/LocalizationManager.cs

```diff
@@ -51,120 +51,13 @@ public class LocalizationManager : ILocalizationManager
             _logger = logger;
         }
 
-        /// <summary>
-        /// Loads all resources into memory.
-        /// </summary>
-        /// <returns><see cref="Task" />.</returns>
-        public async Task LoadAll()
```

Comment:
Why move these around?

Reply:
I have no idea what I wanted to do here so I reverted the commit and made some small improvements